javascript/javascript1/week2/flight_Booking_Fullname_Function.js

```diff
@@ -0,0 +1,41 @@
+/* ________________________ */
+
+function GetFullname(firstname,surname,useFormalName,gender)
+{
+    if(firstname, surname, useFormalName === true)
```

Comment:
this code is not correct in javascript...if you run your program this line would fail.
```suggestion
    if(firstname, surname, useFormalName === true)
```